resources/definitions/fdmprinter.def.json

```
@@ -6069,26 +6069,33 @@
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
-                "brim_outside_only":
+                "brim_location":
                 {
-                    "label": "Brim Only on Outside",
-                    "description": "Only print the brim on the outside of the model. This reduces the amount of brim you need to remove afterwards, while it doesn't reduce the bed adhesion that much.",
-                    "type": "bool",
-                    "default_value": true,
+                    "label": "Brim Location",
+                    "description": "Print a brim on the outside of the model, inside, or both. Depending on the model, this helps reducing the amount of brim you need to remove afterwards, while ensuring a proper bed adhesion.",
+                    "type": "enum",
+                    "options":
+                    {
+                        "outside": "Outside Only",
+                        "inside": "Inside Only",
+                        "everywhere": "Everywhere"
+                    },
+                    "default_value": "outside",
                     "enabled": "resolveOrValue('adhesion_type') == 'brim'",
                     "limit_to_extruder": "skirt_brim_extruder_nr",
                     "settable_per_mesh": false,
                     "settable_per_extruder": true
                 },
                 "brim_inside_margin":
                 {
-                    "label": "Brim Inside Avoid Margin",
-                    "description": "A part fully enclosed inside another part can generate an outer brim that touches the inside of the other part. This removes all brim within this distance from internal holes.",
+                    "label": "Brim Avoid Margin",
+                    "description": "When growing the brim around a surface, it may touch an other surface where brim is not expected. This removes all brim within this distance from clean surfaces.",
```

I think the part "when growing the brim around a surface" is a bit too technical and only makes sense when you know the implementation of a brim, I would simplify the text to
```suggestion
                    "description": "A brim around a surface may touch an other surface where brim is not expected. This removes all brim within this distance from brimless surfaces.",
```